File: MyApplication4/app/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.ImageView;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private EditText name;
    private Button submit;
    private TextView result;
    ImageView image;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        name = findViewById(R.id.editText);
        submit = findViewById(R.id.submit);
        result = findViewById(R.id.result);
        image = findViewById(R.id.imageView);

        //image.setImageResource(R.drawable.ic_launcher_background);

        submit.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v){
                String username = name.getText().toString();
                result.setText(username);
                image.setImageResource(R.drawable.animal_animal_photography_cat_feline_kitten_paws_pet_tabby_4896x3010);
            }
        });

    }
}

File: MyApplication2/app/src/main/java/com/example/myapplication2/MainActivity.kt
package com.example.myapplication2

import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.Surface
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.tooling.preview.Preview
import com.example.myapplication2.ui.theme.MyApplication2Theme
import java.nio.file.attribute.AclFileAttributeView


class MainActivity : ComponentActivity() {

    lateinit var name: EditText
    lateinit var submit: Button
    lateinit var result: TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main_activity);

        name = findViewById(R.id.editText)
        submit = findViewById(R.id.button)
        result = findViewById(R.id.results)

        submit.setOnClickListener {
            var username : String
            username = name.text.toString()
            result.setText(username)
        }
}

}
